src/PatchOrchestrationApplication/Common/WindowsUpdateOperationResult.cs

```diff
@@ -80,7 +87,7 @@ public enum WuOperationResult
     [DataContract]
     public class WindowsUpdateOperationResult
     {
-        public WindowsUpdateOperationResult(String nodeName, DateTime operationTime, WuOperationResult operationResult, IList<WindowsUpdateDetail> updateDetails, WindowsUpdateOperationType operationType, string windowsUpdateQuery, string windowsUpdateFrequency, bool rebootRequired)
+        public WindowsUpdateOperationResult(String nodeName, DateTime operationTime, WuOperationResult operationResult, IList<WindowsUpdateDetail> updateDetails, WindowsUpdateOperationType operationType, string windowsUpdateQuery, string windowsUpdateFrequency, bool rebootRequired, DateTime operationStartTime)
```

Comment:
>operationTime [](start = 70, length = 13)

Update the parameter name in constructor to just operationCompletionTime for clarity and may be have both parameters one after another.

Reply:
Renaming a data member is a breaking change.

Reply:
I am just asking to rename the parameter in the constructor.

---
In reply to: [282464855](https://github.com/microsoft/Service-Fabric-POA/pull/15#discussion_r282464855) [](ancestors = 282464855)